scripts/run_prod.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -x
+
+# Application is configured to run under Supervisord, which in turn monitors
+# the Gunicorn web server
+supervisord -c /code/config/prod/supervisord.conf
```

Comment:
Just an idea, we could add a `-n` here to make supervisord run in the foreground since it is a container. Might make it easier to debug startup problems.